fuzzbucket/reaper.py

```diff
@@ -1,23 +1,37 @@
-import os
-
-from . import list_vpc_boxes, log, get_ec2_client, utcnow
+import typing
 
 from botocore.exceptions import ClientError
 
+import fuzzbucket.cfg as cfg
+
+from . import get_ec2_client, get_vpc_id, list_vpc_boxes, utcnow
+from .log import log
+
+DEFAULT_TTL = float(
+    typing.cast(
+        str,
+        cfg.get("FUZZBUCKET_DEFAULT_TTL", default=str(3600 * 4)),
+    )
+)
+
 
-def reap_boxes(_, __, ec2_client=None, env: dict | None = None) -> dict[str, list[str]]:
+def reap_boxes(_, __, ec2_client=None) -> dict[str, list[str]]:
     ec2_client = ec2_client if ec2_client is not None else get_ec2_client()
-    env = env if env is not None else dict(os.environ)
 
     reaped_instance_ids = []
-    for box in list_vpc_boxes(ec2_client, env["CF_VPC"]):
+    for box in list_vpc_boxes(
+        ec2_client,
+        get_vpc_id(
+            get_ec2_client(),
```

Comment:
perhaps a comment why you are not using `ec2_client` variable?

Reply:
Oh that's a whoopsie. Thank you!